sdk/python/kfp/components/component_factory.py

```diff
@@ -62,9 +62,7 @@ def __getattr__(
                .InputPathPlaceholder, placeholders.OutputUriPlaceholder,
                placeholders.OutputPathPlaceholder]:
         if _name not in ['uri', 'path']:
-            raise AttributeError(
-                'Accessing artifact attribute other than uri or path is not supported.'
-            )
+            raise AttributeError('Cannot access artifact attribute "{_name}".')
```

Comment:
Missing f in f-string. Consider using `self.assertRaisesRegex` in your test code to check that this string is constructed correctly.